.gitmodules

```diff
@@ -1,3 +1,9 @@
 [submodule "src/libbpf"]
 	path = libbpf
 	url = https://github.com/libbpf/libbpf.git
+[submodule "elfutils"]
+	path = elfutils
+	url = git://sourceware.org/git/elfutils.git
```

Comment:
nit: I'd prefer to use the HTTP transport, like `https://sourceware.org/git/elfutils.git`

https://sourceware.org/git/?p=elfutils.git;a=summary

Reply:
After changing this to url = https://sourceware.org/git/elfutils.git

I experience the following issue
```

$ cargo update
    Updating git repository `https://github.com/mmullin-halcyon/libbpf-sys.git`
    Updating git submodule `https://sourceware.org/git/elfutils.git`
warning: spurious network error (3 tries remaining): early EOF; class=Net (12); code=Eof (-20)
warning: spurious network error (2 tries remaining): early EOF; class=Net (12); code=Eof (-20)
warning: spurious network error (1 tries remaining): early EOF; class=Net (12); code=Eof (-20)
error: failed to get `libbpf-sys` as a dependency of package `ttt v0.1.0 (/code/ttt)`

Caused by:
  failed to load source for dependency `libbpf-sys`

Caused by:
  Unable to update https://github.com/mmullin-halcyon/libbpf-sys.git?branch=story/vendorize_dependencies

Caused by:
  failed to update submodule `elfutils`

Caused by:
  failed to fetch submodule `elfutils` from https://sourceware.org/git/elfutils.git

Caused by:
  network failure seems to have happened
  if a proxy or similar is necessary `net.git-fetch-with-cli` may help here
  https://doc.rust-lang.org/cargo/reference/config.html#netgit-fetch-with-cli

Caused by:
  early EOF; class=Net (12); code=Eof (-20)
```

I have tested this on multiple machines by doing the following

$ cargo new ttt && echo "libbpf-sys = { git = \"https://github.com/mmullin-halcyon/libbpf-sys.git\", branch = \"story/vendorize_dependencies\" }" >> ttt/Cargo.toml && cd ttt && cargo build

using ```CARGO_NET_GIT_FETCH_WITH_CLI=true cargo update``` can fix this problem, but it's not a good experience.

I think it might be a better experience to go back to the git://sourceware.org/git/elfutils.git

Reply:
Hmm, weird. It can stay as-is, then.

Reply:
Or rather, it looks like you changed it to `https`. Feel free to change it back to `git`.

Reply:
done